Libraries/Components/Touchable/TouchableNativeFeedback.js

```diff
@@ -94,6 +94,11 @@ type Props = $ReadOnly<{|
    */
   validKeysUp?: ?Array<string>,
 
+  /**
+   * The react tag of the view that follows the current view in the key view loop.
+   */
+  nextKeyViewTag?: ?number, // TODO(macOS GH#768)
+
```

Comment:
Re: Adding this prop to touchable. Looking at the React Native Docs, it seems Touchable will just inherit all the View Props in RN 0.64. Therefore, I think we should either just not set it on Touchable altogether, or revisit this when we update RN-macOS to 0.64 and remove the prop from components that inherit View props.